packages/spor-alert-react-native/src/Alert.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  composeRestyleFunctions,
+  createVariant,
+  spacing,
+  SpacingProps,
+  spacingShorthand,
+  SpacingShorthandProps,
+  useRestyle,
+  useTheme,
+  VariantProps,
+} from "@shopify/restyle";
+import {
+  AltTransportOutline24Icon,
+  CloseOutline18Icon,
+  DeleteCircleOutline24Icon,
+  DropdownDownFill18Icon,
+  DropdownUpFill18Icon,
+  InformationOutline24Icon,
+  SuccessOutline24Icon,
+  WarningOutline24Icon,
+} from "@vygruppen/spor-icon-react-native";
+import React, { useEffect, useState } from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { Theme } from "@vygruppen/spor-theme-react-native";
+import { Text } from "@vygruppen/spor-typography-react-native";
+import { Button } from "@vygruppen/spor-button-react-native";
+import { Pressable } from "react-native";
+
+type Variant = VariantProps<Theme, "alertVariant", "variant">;
+const variant = createVariant({ themeKey: "alertVariant" });
+
+type RestyleProps = SpacingProps<Theme> &
+  SpacingShorthandProps<Theme> &
+  Variant;
+
+const restyleFunctions = composeRestyleFunctions<Theme, RestyleProps>([
+  spacing,
+  spacingShorthand,
+  variant,
+]);
+
+type AlertVariant =
+  | "alternativ-transport"
+  | "important"
+  | "short-transition"
```

Comment:
Skal dette bety "oops her er det kort tid mellom overgangene"? I så fall tror jeg "short-transfer" kan være et bedre navn.

Reply:
Jeg er også litt usikker på om vi burde binde oss såpass opp mot domenet til salgsappen her – det gjør det muligens vanskelig å gjenbruke i andre deler av organisasjonen.

Reply:
godt poeng, jeg har foreløpig endret det til colorscheme⭐️ Per nå gis komponentet et icon utfra colorscheme. Er det da også lurt å skille ut icon fra colorscheme for å få til mer gjenbruk? Eller gir det et for stort variasjonsrom å la bruker selv sende med icon?

Reply:
Gode spørsmål! Jeg er litt usikker på hva som er best approach her egentlig. 

Jeg tror det er bedre at Spor inneholder så generelle komponenter som mulig, og at man heller kan lage mer spesifikke abstraksjoner på toppen senere. Dvs at man må sende inn egne ikoner, og spesifisere fargene som "yellow", "orange" osv osv.